cmd/kubectl-direct_csi/install.go

```diff
@@ -87,7 +87,76 @@ func install(ctx context.Context, args []string) error {
 		glog.Infof("'%s' namespace created", utils.Bold(identity))
 	}
 
-	if err := installer.CreateConversionDeployment(ctx, identity, image, dryRun, registry, org); err != nil {
+	if admissionControl {
+		caCertBytes, publicCertBytes, privateKeyBytes, certErr := installer.GetCerts([]string{installer.AdmissionWebhookDNSName})
+		if certErr != nil {
+			return certErr
+		}
+
+		if err := installer.CreateAdmissionWebhookControllerSecret(
+			ctx,
+			identity,
+			publicCertBytes,
+			privateKeyBytes,
+			dryRun); err != nil {
+			if !errors.IsAlreadyExists(err) {
+				return err
+			}
+		}
+
+		labels := installer.NewObjMeta(identity, identity, "component", "controller").Labels
+		if err := installer.CreateAdmissionWebhookControllerService(ctx, labels, identity, dryRun); err != nil {
+			return err
+		}
+
+		if err := installer.RegisterAdmissionWebhookValidationRules(ctx, identity, caCertBytes, dryRun); err != nil {
```

Comment:
I think the webhook endpoint/service (controller) has to be "live" before registering the webhook config here. Otherwise, the webhook config would be treated as invalid.